model_analyzer/record/metrics_manager.py

```diff
@@ -140,6 +144,7 @@ def profile_server(self):
 
         cpu_only = (not numba.cuda.is_available())
         self._start_monitors(cpu_only=cpu_only)
+        time.sleep(self._config.duration_seconds)
```

Comment:
Does this mean that previously we were not waiting enough for collecting Server only metrics?

Reply:
There were intermittent failures. This became apparent when switching to the remote monitor.